tests/unit/multipleBackend/BackendInfo.js

```diff
@@ -33,26 +34,24 @@ describe('BackendInfo class', () => {
             assert((res.description).indexOf('Bucket ' +
             'Location Error') > -1);
         });
-        it('should return object with applicable error if requestEndpoint ' +
+        it('should return isValid if requestEndpoint ' +
         'is invalid, no objectLocationConstraint or bucketLocationConstraint' +
-        'was provided and data backend is set to "scality"', () => {
+        'was provided, data backend is set to "scality" and legacy location ' +
```

Comment:
shouldn't it be data backend set to multiple (rather than scality)?

Reply:
I have a test with multiple right after this one.
`config.backends.data = 'scality'` is the same as `config.backends.data = 'multiple'`
https://github.com/scality/S3/blob/master/lib/Config.js#L762